assets/stylesheets/components/_global-footer.scss

```diff
@@ -0,0 +1,9 @@
+.global-footer {
+  background-color: $light-grey;
+  border-top: 1px solid tint($light-grey, 80%);
+}
+
+  .global-footer--inner {
+    @extend %content-block;
+    padding: 24px 0 48px; // or some multiple of our line height var
```

Comment:
Rather than this comment shall we add `// TODO` to components where we know we need to revisit the spacing once we've defined our units?